testsuite/tests/quotation/quotation_eval.ml

```diff
@@ -0,0 +1,46 @@
+(* TEST
```

Comment:
Consider adding a test that shows what happens if you write `[%eval: 'a -> 'a]` or `[%eval: 'a]`.